File: Java-Tutorial/Beginner-Friendly-Problems/helloWorld.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Hello World!");

        System.out.print("Enter your full name: ");
        String name = scanner.nextLine(); // Read the entire line, including spaces

        System.out.println("Your name is: " + name);

        scanner.close();
    }
}

File: Cpp/Beginner/helloWorld.cpp
#include <iostream>
#include <string> // For std::string and getline

int main() {
    std::string name; // Allocate memory for the string
    std::cout << "Hello World!" << std::endl;

    std::cout << "Enter your full name: ";
    std::getline(std::cin, name); // Read the entire line, including spaces

    std::cout << "Your name is: " << name << std::endl;
    return 0;
}
